data/Domains.md

```diff
@@ -16,3 +16,4 @@
 | [proj.at & proj.sbs](https://github.com/proj-at/subdomains) | Get your own proj.sbs / proj.at subdomains, free for open-source developers. | ❌ |
 | [thedev.id](https://thedev.id) | A free thedev.id subdomain for developers. | ❌ |
 | [thedev.me](https://thedev.me) | Free {name}.thedev.me domain for projects and personal sites. | ❌ |
+| [is-not.tech](https://example.com) | Grab yourself a free is-not.tech subdomain for tech enthusiasts or anyone who is new to web development! | 
```

Comment:
You need to update the link from example.com and you also need to sort it alphabetically.

However due to is-not-tech/register#13 I don't think I will be able to approve this as it poses a huge security risk for attackers updating users DNS records to point to spam/scam websites.